suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -144,15 +144,14 @@ export const prepareAccountsReducer = createReducerWithExtraDeps(
 
 export const selectAccounts = (state: AccountsRootState) => state.wallet.accounts;
 
-export const selectAccountsByDeviceState = memoizeWithArgs(
-    (state: AccountsRootState, deviceState: string): Account[] =>
-        pipe(
-            selectAccounts(state),
-            A.filter(account => account.deviceState === deviceState),
-        ) as Account[],
-    // cache up to 3 devices to make sure it works correctly
-    { size: 3 },
-);
+export const selectAccountsByDeviceState = (
```

Comment:
Without memoization it feels that switching between home, assets and receive tab takes really long time. In my ios simulator it took 7 seconds on fully discovered allseed.

Reply:
Hmm. Now i tried the same on different branch and it also took a long time 🤔. So maybe not related to this change, but still feels very slow.

Reply:
Good to know I will check what's happening.

Reply:
I think it's because there is quite a lot of accounts on all seed and there is no FlashList for Accounts/Receive screen.

Reply:
not related to any of my changes